src/coreclr/jit/assertionprop.cpp

```diff
@@ -3835,37 +3835,16 @@ GenTree* Compiler::optAssertionProp_LclFld(ASSERT_VALARG_TP assertions, GenTreeL
             break;
         }
 
-        // See if the variable is equal to another variable or a constant.
+        // See if the variable is equal to another variable.
```

Comment:
Strange, I thought I had already removed this.

Also `fgMorphTree` already iterates, assuming there are opportunities to chain assertions.